daemon/call.c

```diff
@@ -2123,7 +2123,7 @@ static void __dtls_logic(const sdp_ng_flags *flags,
 		__fingerprint_changed(other_media);
 		other_media->fingerprint = sp->fingerprint;
 	}
-	else if (other_media->tls_id.len && (sp->tls_id.len || str_cmp_str(&other_media->tls_id, &sp->tls_id))) {
```

Comment:
I think this should actually be `(other_media->tls_id.len && (sp->tls_id.len == 0 || str_cmp_str(&other_media->tls_id, &sp->tls_id)))`

Reply:
I will test this in my environment.

Reply:
Hi !

I have tested changes, no errors, so i push it to this PR.